webapp/components/ContentMenu.vue

```diff
@@ -1,32 +1,13 @@
 <template>
-  <dropdown
-    class="content-menu"
-    :placement="placement"
-    offset="5"
-  >
-    <template
-      slot="default"
-      slot-scope="{toggleMenu}"
-    >
-      <slot
-        name="button"
-        :toggleMenu="toggleMenu"
-      >
-        <ds-button
-          class="content-menu-trigger"
-          size="small"
-          ghost
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-05-15T14:48:57Z" title="Wednesday, May 15th 2019, 4:48:57 pm +02:00">May 15, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -98,15 +98,39 @@ export default {
           icon: 'flag'
         })
       }
-
+console.log(this);
       if (!this.isOwner && this.isModerator) {
-        routes.push({
-          name: this.$t(`disable.${this.resourceType}.title`),
-          callback: () => {
-            this.openModal('disable')
-          },
-          icon: 'eye-slash'
-        })
+        //console.log(this.resource.disabled);
+
+        if (!this.resource.disabled) {
+
+          // Beitrag ist freigegeben
```


@ogerly remove code comments. Especially remove them if they are in German.